src/stko/_internal/optimizers/optimizers.py

```diff
@@ -117,6 +120,81 @@ def optimize(self, mol: MoleculeT) -> MoleculeT:
         return mol
 
 
+class OptWriterSequence(Optimizer):
+    """Applies optimizers in sequence and writes each step to avoid reruns.
+
+    Parameters:
+        optimizers:
+            A number of optimizers, each of which gets applied to a
+            molecule, based on the order given.
+
+    Examples:
+        Let's say we want to embed a molecule with ETKDG first and then
+        minimize it with the MMFF force field.
+
+        .. code-block:: python
```

Comment:
this example is not being tested -- dangerous

Reply:
I will fix this, ty!